extensions/gdx-setup/src/com/badlogic/gdx/setup/Dependency.java

```diff
@@ -37,4 +37,13 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Always override `hashCode()` when you override `equals()`, here is [why](http://www.xyzws.com/javafaq/why-always-override-hashcode-if-overriding-equals/20)